docs/src/includes/_types.md

```diff
@@ -418,8 +418,8 @@ Stores static common data for an [ENT_TYPE](#ENT_TYPE). You can also clone entit
 
 Type | Name | Description
 ---- | ---- | -----------
-[EntityDB](#EntityDB) | [new(EntityDB other)](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=EntityDB) | 
-[EntityDB](#EntityDB) | [new(ENT_TYPE)](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=EntityDB) | 
+[EntityDB](#EntityDB) | [new(EntityDB& other)](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=EntityDB) | 
+[EntityDB](#EntityDB) | [new(ENT_TYPE other)](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=EntityDB) | 
```

Comment:
did i accidentally fix something?
or break something?

Reply:
Looks like a fix to me...